storage_test.go

```diff
@@ -245,22 +241,18 @@ func TestStorageSnapshots_ListStorageSnapshots(t *testing.T) {
 
 	expected := []Snapshot{
 		{
-			Region:        &Region{Slug: "nyc3"},
+			Regions:       []string{"nyc3"},
 			ID:            "80d414c6-295e-4e3a-ac58-eb9456c1e1d1",
-			VolumeID:      "98d414c6-295e-4e3a-ac58-eb9456c1e1d1",
 			Name:          "my snapshot",
-			Description:   "my description",
 			SizeGigaBytes: 100,
-			CreatedAt:     time.Date(2002, 10, 02, 15, 00, 00, 50000000, time.UTC),
+			Created:       "2002-10-02T15:00:00.05Z",
```

Comment:
Hm, looks like we're mixed between strings and `time.Time` and `Created` vs. `CreatedAt`:
 * https://github.com/digitalocean/godo/blob/2ff8a02a86cd6918b384a5000ceebe886844fbce/storage.go#L56
* https://github.com/digitalocean/godo/blob/5c91372eb44e557d4f3c52a7e1c2c96fdf636673/images.go#L39
* https://github.com/digitalocean/godo/blob/c3a2aad75d576aed8f14dcaea896e40eff83dd18/droplets.go#L55

I can do another PR to standardize the others, what do you think of settling on `Created` as a `time.Time`?